src/audio/CMakeLists.txt

```diff
@@ -8,9 +8,6 @@ if(NOT CONFIG_LIBRARY)
 		buffer.c
```

Comment:
Has it been tested in testbench that this change is bit exact with old volume version? I assume module API does not change the data, it's just higher abstraction level for components. Should test with topology with a fairly long volume ramp length with zero crossings check both disabled and enabled.